builtin/aws/lambda/lambda.go

```diff
@@ -5,10 +5,11 @@ import (
 	sdk "github.com/hashicorp/waypoint-plugin-sdk"
 )
 
-//go:generate protoc -I ../../../.. --go_opt=plugins=grpc --go_out=../../../.. waypoint/builtin/aws/lambda/plugin.proto
+//go:generate protoc -I ../../../.. -I ../../../thirdparty/proto/opaqueany --go_out=../../../.. waypoint/builtin/aws/lambda/plugin.proto
```

Comment:
I didn't touch the `waypoint` repo for sometime, and this `go:generate` directive stopped working for me.

Running this...
```
//go:generate protoc -I ../../../.. --go_opt=plugins=grpc --go_out=../../../.. waypoint/builtin/aws/lambda/plugin.proto
```
...outputted this
```console
[Info  - 9:42:00 AM] 2022/04/07 09:42:00 protoc -I ../../../.. --go_opt=plugins=grpc --go_out=../../../.. waypoint/builtin/aws/lambda/plugin.proto

	operation="generate"

[Info  - 9:42:00 AM] 2022/04/07 09:42:00 --go_out: protoc-gen-go: plugins are not supported; use 'protoc --go-grpc_out=...' to generate gRPC

See https://grpc.io/docs/languages/go/quickstart/#regenerate-grpc-code for more information.

	operation="generate"

[Info  - 9:42:00 AM] 2022/04/07 09:42:00 lambda.go:10: running "protoc": exit status 1

	operation="generate"

[Error - 9:42:00 AM] Request workspace/executeCommand failed.
  Message: exit status 1
  Code: 0 
[Error - 9:42:00 AM] 2022/04/07 09:42:00 command error: exit status 1

[Error - 9:42:20 AM] Request textDocument/definition failed.
  Message: no identifier found
  Code: 0 
[Error - 9:42:20 AM] Request textDocument/definition failed.
  Message: no identifier found
  Code: 0 
```